core/frontend/src/render/webgl/BranchUniforms.ts

```diff
@@ -70,6 +70,8 @@ export class BranchUniforms {
   private readonly _scratchVIModelMatrix = Transform.createIdentity();
   private readonly _zeroPoint = new Point3d(0, 0, 0);
 
+  public overrideClipStyle = false;
```

Comment:
This is really `disableClipStyle`, isn't it? What led you to decide to use a boolean checked in the shader, vs moving ClipStyle into BranchState?

Reply:
Since the bug this was trying to solve was from a mis-application of the clip style, passing a single boolean along seemed to be the most efficient way to solve the bug. That being said, it does limit the application of this to only solving the related bug and other very similar scenarios where we wanted specifically no ClipStyle. If we wanted a solution that could be applied to more in the future, especially if we wanted to allow multiple ClipStyles per viewport, then adding ClipStyle to BranchState would be a better solution.

Reply:
As of the latest commit, `overrideClipStyle` has been renamed to `disableClipStyle`. The logic for using this boolean has been moved to within `PushBranch` in `BranchUniforms` and has been removed from the shader. Now, in `PushBranch`, the alpha values of `ClipStack.insideColor`, `ClipStack.outsideColor`, and `ClipStack.intersectionStyle` are set to 0 if `disableClipStyle` is true, thereby disabling them. Otherwise, it is left to the viewport's clip style to decide the alpha values.